intro-HPC/ch_introduction.tex

```diff
@@ -351,6 +377,25 @@ \subsection{What operating systems can I use?}
 A user does not need to have prior knowledge about Linux; all of the required
 knowledge is explained in this tutorial.
 
+\subsection{What does a typical workflow look like?}
+
+A typical workflow looks like:
+
+\begin{enumerate}
+    \item Connect to the login nodes with SSH (see \autoref{sec:first-time-connection-to-the-hpc})
+    \item Transfer your files to the cluster (see \autoref{sec:filetransfer})
+    \item Optional: compile your code and test it (for compiling, see \autoref{ch:compiling-and-testing-your-software-on-the-hpc})
+    \item Create a job script and submit your job (see \autoref{ch:running-batch-jobs})
+    \item Get some coffee and be patient:
+    \begin{enumerate}
+        \item Your job gets into the queue
+        \item Your job gets executed
+        \item Your job finishes
+    \end{enumerate}
+    \item Move your results from the cluster to your computer.
```

Comment:
Study the results generated by your jobs, either on the cluster or after downloading them locally.